lavalink.py

```diff
@@ -94,7 +97,9 @@ def unregister_listeners(self):
                 print('[Lavalink.py] Attempting to reconnect (Attempt: {})'.format(a + 1))
                 await self._connect()
                 # if connection has been established, stop trying
-                if self.bot.lavalink.ws.open:
+                if not self.bot.lavalink.ws.open:
+                    continue
+                else:

```

Comment:
This else isn't needed, as `continue` will automatically pass the current iteration thus no further code is run.

Reply:
As I understand it, without the else, you'll still see the error message even after a successful reconnect.

Reply:
I've seen no such behavior without the else, although I will quickly test

Reply:
https://i.crimsonxv.pro/24b76ab345.png

Reply:
You didn't see it because before it was simply a return. The only thing this change made was take the implicitness out of the equation.

Reply:
In all honesty, this change is a little unneeded. I'd be willing to remove it.

Reply:
Eh, may as well keep it. Will merge.